idea/src/org/jetbrains/kotlin/idea/intentions/ConvertCollectionConstructorToFunction.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.idea.intentions
+
+import com.intellij.openapi.editor.Editor
+import org.jetbrains.kotlin.psi.KtNameReferenceExpression
+import org.jetbrains.kotlin.psi.KtPsiFactory
+
+class ConvertCollectionConstructorToFunction : SelfTargetingIntention<KtNameReferenceExpression>(
+    KtNameReferenceExpression::class.java, "Convert Java collection call to function call"
+) {
+
+    private val functionMap = hashMapOf(
+        "ArrayList" to "arrayListOf",
+        "HashMap" to "hashMapOf",
+        "HashSet" to "hashSetOf"
```

Comment:
`LinkedHashMap`, `LinkedHashSet`

https://docs.oracle.com/javase/8/docs/api/java/util/LinkedHashSet.html

https://docs.oracle.com/javase/8/docs/api/java/util/LinkedHashMap.html

The rest are interfaces, I think.

Reply:
I couldn't find a function call for them, you also said it was just those 3 when we went over them, but maybe I overlooked them, I'll look again during the weekend.